huggingface/docker/Dockerfile

```diff
@@ -45,6 +45,6 @@ RUN cd /stage && git clone https://github.com/microsoft/huggingface-transformers
 # Install AzureML support and commonly used packages.
 # pin datasets==1.9.0 due 'Sequence to truncate too short to respect the provided max_length' on roberta-large 
```

Comment:
Add a comment like for "datasets==1.9.0" above explaining the pin.

Reply:
Added comments as suggested